update_projects_table_test.py

```diff
@@ -1,119 +1,132 @@
+"""Unit tests for the "update_projects_table" module."""
+
 from pathlib import Path
-import tempfile
-import unittest
 
+import pytest
 import update_projects_table
 
 
-class TestCreateProjectTable(unittest.TestCase):
-    """Unit tests for the "update_projects_table" module."""
+class TestExtractMetadata:
```

Comment:
Why do we need the class? `self` is not used in any method.

Reply:
clear visual separation between the tests of each function